ReactCommon/fabric/components/root/RootShadowNode.cpp

```diff
@@ -55,11 +60,24 @@ UnsharedRootShadowNode RootShadowNode::clone(
     sharedChildren = std::make_shared<SharedShadowNodeList>(children);
 
     oldChild = ancestor.get().shared_from_this();
-    newChild = oldChild->clone(ShadowNodeFragment{.children = sharedChildren});
+    newChild = oldChild->clone({
```

Comment:
Here and everywhere with `ShadowNodeFragment`:

I think if we should:
 * Rename `ShadowNodeFragment::nullSharedProps()` (and family) to `ShadowNodeFragment::propsPlaceholder()`;
 * Introduce similar functions for the rest of members (tag and rootTag);
 * Feel free to rename `rootTag` to `surfaceId`.

I believe if we go this way, we don't need comments in the code.

Seems this is only data structure worth to improve this way, the rest is fine.